pom.xml

```diff
@@ -70,6 +70,15 @@
     <prometheus.client.version>0.16.0</prometheus.client.version>
     <reflections.version>0.10.2</reflections.version>
 
+    <!-- The `optional` tag in `dependencyManagement` is not inherited; see
+    https://issues.apache.org/jira/browse/MNG-5227 and
+    https://issues.apache.org/jira/browse/MNG-5632. Therefore, we specify
+    optional dependencies here to manage them in a single location. -->
+    <prometheus.client.optional>true</prometheus.client.optional>
+    <com.amazonaws.aws-java-sdk-sts.optional>true</com.amazonaws.aws-java-sdk-sts.optional>
+    <com.google.auth.google-auth-library-oauth2-http.optional>true</com.google.auth.google-auth-library-oauth2-http.optional>
+    <org.springframework.boot.spring-boot-actuator.optional>true</org.springframework.boot.spring-boot-actuator.optional>
```

Comment:
`prometheus.client` covers multiple artifacts so I followed `prometheus.client.version`. Since others cover only single artifact, I used `groupId.artifactId.optional`. Any suggestions for naming here?

Reply:
that's fine.